server/app/models/grid_domain_authorization.rb

```diff
@@ -18,6 +27,22 @@ class GridDomainAuthorization
 
   # @return [String]
   def to_path
-    "#{self.grid.try(:name)}/#{self.name}"
+    "#{self.grid.try(:name)}/#{self.domain}"
   end
+
+  def computed_state
+    deploy = find_deploy
+    if deploy && !deploy.finished?
```

Comment:
Also `error` if `deploy.error?` - ideally the CLI would also be able to show the `deploy.reason` if it fails